src/main/deltachat/webxdc.ts

```diff
@@ -221,10 +227,15 @@ export default class DCWebxdc extends SplitOut {
           },
           title: makeTitle(webxdcInfo, chatName),
           icon: app_icon || undefined,
-          width: 375,
-          height: 667,
+          width: lastBounds.width,
+          height: lastBounds.height,
```

Comment:
```suggestion
```

These can now be removed now, right? [They're optional](https://www.electronjs.org/docs/latest/api/browser-window#new-browserwindowoptions).